scripts/localhost/deployWFAIR.config.json

```diff
@@ -0,0 +1,16 @@
+{
+  "wallfairTotalSupply": "1000000000",
+  "transferRequests" :
+    [
+      {
+        "name": "Wallfair Public Hot Wallet #1",
```

Comment:
where those are coming from? we need to send funds from seed, private and public (30,000,000.00 + 5,000,000.00 + 60,000,000.00) into Sale Hot Wallet and 2,500,000 into DEX Listing

Reply:
They are just test figures during development. The correct configuration of the config files is a separate task.

Reply:
I've created https://wallfair-product.atlassian.net/browse/MVP2-349 to track this.